tools/oauthify/main.go

```diff
@@ -68,7 +68,7 @@ many features, such as:
 	//url := conf.AuthCodeURL("state", oauth2.AccessTypeOffline)
 
 	url := conf.AuthCodeURL("state")
-	fmt.Println("Visit the following URL, then paste the resulting code below:\n")
+	fmt.Println("Visit the following URL, then paste the resulting code below:")
```

Comment:
This one was supposed to be as it was to create a blank line before the following code, making it easier to see. Is it creating some kind of build problem? If so, perhaps some other fix that preserves the original output?

Reply:
This code caused a go vet error. In the latest go version, go vet is also executed by go test.

Reply:
Fixed https://github.com/attic-labs/noms/pull/3815/commits/b77a2257af5ad62c8c256f63e464e0af32df1349